### ISSUE TITLE
iOS and Android idsAvailable return values not consistent

### ISSUE THREAD

**Issue by tonyxiao** [OneSignal/react-native-onesignal#42]
on iOS we return

``` objc
NSDictionary *value = @{
                                    @"pushToken": pushToken,
                                    @"playerId" : userId
                                    };
            callback(@[value]);
```

On Android we return

``` java
final WritableMap value = Arguments.createMap();
                    value.putString("userId", userId);
                    value.putString("pushToken", registrationId);

                    callback.invoke(value);
```

We should return either `userId` or `playerId` and keep it the same on both platforms for consistency

**Comment by avishayil** [comment]
Thanks for the heads-up, i'll take care of it.

**Comment by avishayil** [comment]
Fixed by 1.1.6
